style.css

```diff
@@ -0,0 +1,271 @@
+body {
+  display: grid;
+  grid-template:
+    "header header header" 50px
+    "left-bar main right-bar" 1fr /
+    200px 1fr 200px;
+  height: 100vh;
+  font-family: sans-serif;
+  margin: 0;
+  background: black;
+  color: white;
+  margin-left: 3vw;
+  margin-right: 5vw;
+}
+
+
+/* Header start */
+.header {
+  grid-area: header;
+  display: grid;
+  grid-template: 
+    "img h1 search-bar" 50px /
+   200px 1fr 200px;
+  position: sticky; top: 0;
+}
+
+.header img {
+  width: 60px;
+}
+
+.header h1 {
+  border-left: solid 1px #2F3336;
+  border-right: solid 1px #2F3336;
+  font-size: large;
+  margin-left: 10px;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-top: 10px;
+  padding-left: 10px;
+}
+
+.header .search-bar {
+  display: inline-flex;
+  place-items: center;
+  background: #202327;
+  border-radius: 50px;
+  margin: 10px;
+  width: 225px;
+}
+
+.header .search-bar p {
+  margin: 0;
+  color: #71767B;
+  text-align: center;
+  line-height: 2;
+}
+
+.header .search-bar img {
+  width: 20px;
+  margin: 15px;
+}
+/* header end */
+
+
+/* twitter main start */
+.main {
+  border-top: solid 1px #2F3336;
+  border-left: solid 1px #2F3336;
+  border-right: solid 1px #2F3336;
+  height: 100vh;
+  padding-right: 10px;
+  margin-left: 10px;
+}
+
+.enter-tweet .text-box {
+  display: inline-flex;
+  margin: 20px;
+  gap: 50px;
+  color: #6A6F74;
+  line-height: 60px;
+}
+
+.tweets {
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.tweet-cards {
+  border-top: solid 1px #2F3336;
+  display: grid;
+  grid-template:
+    "pfp1 name tag time" 50px
+    "message message message message" 100px /
+    60px 100px 100px 1fr 100px;
+  gap: 30px;
+  padding-top: 10px;
+}
+
+.pfp1 {grid-area: pfp1;}
+.name {grid-area: name;}
+.tag {grid-area: tag;}
+.time {grid-area: time;}
+.message {grid-area: message;}
+
+.tweet-cards-buttons {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  margin-bottom: 20px;
+}
+
+.attachments {
+  display: inline-flex;
+  gap: 20px;
+  margin-left: 80px;
+  border-top: solid 1px #2F3336;
+}
+
+.attachments img {
+  width: 30px;
+}
+
+.main-tweet-button {
+  background: #FF2819;
+  width: 100px;
+  border-radius: 50px;
+  height: 50px;
+  text-align: center;
+  line-height: 50px;
+  margin-left: 300px;
+  margin-top: 10px;
+}
+
+.tweet-cards-buttons img {
+  width: 30px;
+}
+
+/* twitter main end */
+
+
+/* left bar start */
+.left-bar {
```

Comment:
Making this sticky (or fixed) as a whole could have been a good idea – the left bar shouldn't scroll up behind the twitter logo.